package.json

```diff
@@ -66,6 +68,7 @@
         "eslint-plugin-prettier": "^3.1.3",
         "eslint-plugin-react": "^7.19.0",
         "file-loader": "^6.2.0",
+        "indefinitely-typed": "^1.1.0",
```

Comment:
Same as above

Reply:
This is actually in devDependencies, but it's probably not obvious when code-reviewing because of code collapse.